controllers/tests/controller/selfnoderemediation_controller_test.go

```diff
@@ -158,10 +158,13 @@ var _ = Describe("SNR Controller", func() {
 			isAdditionalSetupNeeded = true
 		})
 
-		Context("ResourceDeletion strategy", func() {
+		Context("Automatic strategy - ResourceDeletion selected", func() {
 
 			BeforeEach(func() {
-				remediationStrategy = v1alpha1.ResourceDeletionRemediationStrategy
+				remediationStrategy = v1alpha1.AutomaticRemediationStrategy
+				prevVal := utils.IsOutOfServiceTaintGA
```

Comment:
NIT: Why do we take the `IsOutOfServiceTaintGA` value instead of the `IsOutOfServiceTaintSupported` value? I believe it won't change the functionality of the unit tests, but it seems more accurate to test the supported version rather than the GA version

Reply:
This change is to allow the test to test AutomaticRemediationStrategy, that strategy is only affected by IsOutOfServiceTaintGA value.